src/core/stream.js

```diff
@@ -740,14 +740,15 @@ var PredictorStream = (function PredictorStreamClosure() {
     var i;
 
     if (bits === 1) {
-      for (i = 0; i < rowBytes; ++i) {
-        var c = rawBytes[i];
-        inbuf = (inbuf << 8) | c;
-        // bitwise addition is exclusive or
-        // first shift inbuf and then add
-        buffer[pos++] = (c ^ (inbuf >> colors)) & 0xFF;
-        // truncate inbuf (assumes colors < 16)
-        inbuf &= 0xFFFF;
+      var complement = 0;
+      var bitOrder = 0;
+      for(i=0; i < rowBytes; ++i) {
+        inbuf = (inbuf << 8) | (rawBytes[i] & 0xFF);
+        for(bitOrder=7; bitOrder>=0; --bitOrder) {
+            complement = (complement + (inbuf >> bitOrder)) & 0x1;
```

Comment:
Remove two spaces from the indenting for this line and the next line. We indent each block with two spaces.